lib/crypto.nit

```diff
@@ -183,6 +183,36 @@ redef class Text
 	end
 end
 
+redef class Bytes
+
+	# Returns `self` xored with `key`
+	#
+	# The shortest of the two is cycled through until the longest has been
+	# completely xored.
```

Comment:
I'm not sure if using the shortest is a POLA approach.
I assumed that xorcrypting was symmetric. ie `message.xor(key).xor(key) == message`. But if key.length > message.length this become false.